### PR TITLE
[camunda][SIMS#577] Fixes for Assessment Calculations

### PR DESCRIPTION
**Pull request by abschwenker**
1. Added institution types to eligiblity
2. Resolved award calculation issues
3. Added compatibility for supporting users

### CODE REVIEW

File: sources/packages/workflow/BPMN/camunda-8/assessment-2023-2024.bpmn [hunk 1]
```diff
@@ -294,9 +294,9 @@
       <bpmn:incoming>Flow_1fd7dws</bpmn:incoming>
       <bpmn:outgoing>Flow_0gv4hzw</bpmn:outgoing>
     </bpmn:businessRuleTask>
-    <bpmn:businessRuleTask id="Activity_0iphb43" name="limitApplicationTransportation">
+    <bpmn:businessRuleTask id="Activity_0iphb43" name="DMNApplicationTransportationMaximum">
       <bpmn:extensionElements>
-        <zeebe:calledDecision decisionId="applicationTransportationMaximum" resultVariable="DMNCalculatedDataApplicationtransportationMaximum" />
+        <zeebe:calledDecision decisionId="DMNApplicationTransportationMaximum" resultVariable="DMNCalculatedDataApplicationtransportationMaximum" />
```
Comment (andrewsignori-aot) [inline sources/packages/workflow/BPMN/camunda-8/assessment-2023-2024.bpmn:299]: Talked with @abschwenker about some variables with acronyms not starting if the lower case. This should not block the PR and we address it later to not delay this effort.

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍 
First time ever that the full-time calculations are happening and they are matching the spreadsheet.

**Review by ann-aot — APPROVED** [review]
👍

File: sources/packages/workflow/BPMN/camunda-8/assessment-decisions.dmn [hunk 2]
```diff
@@ -2371,7 +2371,7 @@
           <text>31868</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1u665kl">
-          <text>0.0238 / 8 * 12 / 52</text>
+          <text>0.0238 * 3 / 13</text>
```
Comment (andrepestana-aot) [inline sources/packages/workflow/BPMN/camunda-8/assessment-decisions.dmn:2374]: What are those values used for? Shouldn't we use constants? They repeat  a lot...

**Review by andrepestana-aot — COMMENTED** [review]
Looks good. I just left some questions.

File: sources/packages/workflow/BPMN/camunda-8/assessment-decisions.dmn (hunk 2 above)
Comment (abschwenker) [reply sources/packages/workflow/BPMN/camunda-8/assessment-decisions.dmn:2374]: the 0.0238 are the values that are provided to us run the assessment. the " * 3 / 13" is a way to convert that value to the "format" used in our weekly calculations.

**Pull request merged**